insights/templates/base.html.j2

```diff
@@ -16,10 +16,18 @@
 
   {% if config.ENV == 'development' %}
   <script src="https://cdn.jsdelivr.net/npm/vue@2/dist/vue.js"></script>
+  <script src="https://d3js.org/d3.v6.js"></script>
+  <script src="https://unpkg.com/@popperjs/core@2/dist/umd/popper.min.js"></script>
+  <script src="https://unpkg.com/tippy.js@6/dist/tippy-bundle.umd.js"></script>
+
   <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.4/Chart.bundle.js"></script>
-  <script src="https://unpkg.com/vue-chartjs@3.5.1/dist/vue-chartjs.min.js"></script>
+  <script src="https://unpkg.com/vue-chartjs@3.5.1/dist/vue-chartjs.js"></script>
   {% else %}
   <script src="https://cdn.jsdelivr.net/npm/vue@2.6.14/dist/vue.min.js" integrity="sha256-kXTEJcRFN330VirZFl6gj9+UM6gIKW195fYZeR3xDhc=" crossorigin="anonymous"></script>
+  <script src="https://d3js.org/d3.v6.js"></script>
+  <script src="https://unpkg.com/@popperjs/core@2"></script>
+  <script src="https://unpkg.com/tippy.js@6"></script>
```

Comment:
Are these duplicate of lines 19-21?

Reply:
It's duplicated due to one section being in the {% if debug ...} in the jinja template, this is so we can load the nonmin versions and the debug version of vuejs when running in development mode.